src/components/ScrollToTop/ScrollToTop.tsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function ScrollToTop() {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pathname]);
+
+  return null;
```

Comment:
if it's necessary to return null from that function?

Reply:
It is necessary to return null, otherwise I will get compilation error.
![image](https://user-images.githubusercontent.com/83533040/147783612-e7fd3987-1131-4c0e-a8df-979c79e0b8b6.png)
I get this code snippet I get from official page of react-router, therefore I believe all is right with this component…
![image](https://user-images.githubusercontent.com/83533040/147783862-040d9545-8e90-4ff2-acf2-8cf8a3089f1c.png)